docs/DeveloperGuide.md

```diff
@@ -52,6 +52,47 @@ in Bookmark mode.
 
 
 
+**Academic Component**
```

Comment:
can change to ##Academic Component!

Reply:
I'll merge and help edit this since I'm working on it, thanks for pointing!